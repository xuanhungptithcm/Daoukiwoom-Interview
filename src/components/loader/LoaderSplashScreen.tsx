import { ReactElement } from "react";
import './loader-splash.scss';

const LoaderSplashScreen = (): ReactElement => (
  <div>
    <div className="lds-dual-ring"></div>
  </div>
);

export default LoaderSplashScreen;
