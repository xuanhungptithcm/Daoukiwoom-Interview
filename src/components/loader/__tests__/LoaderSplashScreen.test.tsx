/**
 * @jest-environment jsdom
 */

import { shallow } from "enzyme";
import LoaderSplashScreen from "../LoaderSplashScreen";

describe("LoaderSplashScreen", () => {
  test("should properly render", () => {
    expect(shallow(<LoaderSplashScreen />)).toMatchSnapshot();
  });
});
