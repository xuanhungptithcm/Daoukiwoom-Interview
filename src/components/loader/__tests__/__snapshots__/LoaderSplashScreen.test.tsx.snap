// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoaderSplashScreen should properly render 1`] = `
<div>
  <div
    className="lds-dual-ring"
  />
</div>
`;
